kubernetes/policies/general/uses_image_tag_latest_test.rego

```diff
@@ -58,3 +58,62 @@ test_tagged_image_allowed {
 
 	count(r) == 0
 }
+
+test_tagged_image_with_digest_allowed {
+	r := deny with input as {
+		"apiVersion": "v1",
+		"kind": "Pod",
+		"metadata": {"name": "hello-tag"},
+		"spec": {"containers": [{
+			"command": [
+				"sh",
+				"-c",
+				"echo 'Hello' && sleep 1h",
+			],
+			"image": "busybox:1.33.1@sha256:askj78jhkf278hdjkf78623gbkljmkvmk8kjn98237487hkjaf897bkjsehf783f",
+			"name": "hello",
+		}]},
+	}
+
+	count(r) == 0
+}
+
+test_tagged_image_with_digest_denied {
```

Comment:
These 2 cases maybe should pass; even though `latest` (or no tag) is used, it's pinned to a concrete digest.  If you agree, I can update to allow these cases.

Reply:
yeah I agree, see my other comment

Reply:
Rule updated and test changed to allow this case.